src/main/java/model/user/User.java

```diff
@@ -3,14 +3,46 @@
 import entity.BaseEntity;
 import entity.Printable;
 
+import java.sql.Timestamp;
+import java.util.UUID;
+
 public class User extends BaseEntity implements Printable {
+    private String name;
+    private Timestamp creationDate;
+
+    public User() {
+
+    }
+
+    public User(UUID id, String name, Timestamp creationDate) {
+        this.id = id;
```

Comment:
do not define id explicitly, generate it in BaseEntity or on db level

Reply:
Generated id in BaseEntity level